scripts/fuzz_opt.py

```diff
@@ -1391,6 +1391,108 @@ def handle(self, wasm):
         compare_between_vms(output, merged_output, 'Merge')
 
 
+# Tests wasm-split
+class Split(TestCaseHandler):
+    frequency = 1  # TODO: adjust lower when we actually enable this
+
+    def handle(self, wasm):
+        # get the list of function names, some of which we will decide to split
+        # out
+        wat = run([in_bin('wasm-dis'), wasm] + FEATURE_OPTS)
+        all_funcs = re.findall(r'\n [(]func [$](\S+)', wat)
+
+        # get the original output before splitting
+        output = run_d8_wasm(wasm)
+        output = fix_output(output)
+
+        # find the names of the exports. we need this because when we split the
+        # module then new exports appear to connect the two halves of the
+        # original module. we do not want to call all the exports on the new
+        # primary module, but only the original ones.
+        exports = []
```

Comment:
It might be simpler to use the `--export-prefix` option to add a unique prefix to the new exports that we can filter out directly in the JS wrapper.

Reply:
Hmm, then we'd need to hardcode some special prefix in the JS wrapper, but I'm not sure there is a fixed prefix we can use: we don't want to overlap with existing export names.

Reply:
FWIW Emscripten uses `%` as the prefix. Maybe we can just choose an arbitrary one like that and it would be good enough?

Reply:
I think the difference is that Emscripten has full control over the export names. In the fuzzer we do want to be able to fuzz initial content from anywhere. I suppose we could sanitize that content before fuzzing it, but that seems more complicated to me.

Reply:
Ok, the currently solution LGTM, then.